### ISSUE TITLE
(Suggestion) Java Profile Support

### ISSUE THREAD

**Issue by unknowntrojan** [MultiMC/MultiMC5#3608]
System Information
-----------------------------
MultiMC version: newest develop

Operating System: Windows 10

Summary of the issue or suggestion:
----------------------------------------------
Currently, for Minecraft, a few versions of Java are better/worse for certain workloads.
For example, I have to juggle around parameters and java paths every time I create an instance, since I have a good OJ9 config, Hotspot config with a few different GCs and a GraalVM config for every installed java version I have. I change these a lot dependent on what I'm doing.
Maybe something such as "Java Profiles" might be in order?
These would include the java path, arguments, and maybe memory allocation (i.e. aikars openj9 guide requires tuning launch params in accordance to your memory allocation).

This would also help benchmarking your instances to see which configuration you prefer.

( I hope the Steps to reproduce don't need to be filled here )

What should happen:
------------------------------
Create an instance, select a fitting java profile and play.

**Comment by Zetabite** [comment]
nah, that just adds another layer of problems that can be easier solved by just adding manually. people already have issues using the normal menu, adding a profile menu wont make it easier

**Comment by unknowntrojan** [comment]
Well we don't need to remove the old global/override system for Java, it could just be a setting.

**Comment by wb1016** [comment]
had same thought. good suggestion.
